test/markdown_test.rb

```diff
@@ -126,6 +126,12 @@ def test_whitespace_after_urls
     assert_equal exp, rd
   end
 
+  def test_auto_linked_email_utf8_issue
+		rd = render_with({ autolink: true }, "a@b.c」\nd@example.coü")
```

Comment:
This part fails on the latest source at 03b664da. So it should be:

```rb
rd = render("a@b.c」\nd@example.coü", with: [:autolink])
exp = %{<p><a href="mailto:a@b.c">a@b.c</a>」\n<a href="mailto:d@example.co">d@example.co</a>ü</p>}
```

Reply:
OMG this project is still alive!!

Got it, I will rebase this PR soon. Thanks for reviewing. :)